service/internal/cloudconfig/tcnp.go

```diff
@@ -92,6 +94,36 @@ func (t *TCNP) NewTemplates(ctx context.Context, obj interface{}) ([]string, err
 		cl = list.Items[0]
 	}
 
+	var md apiv1alpha3.MachineDeployment
+	{
+		var list apiv1alpha3.MachineDeploymentList
+		err := t.config.K8sClient.CtrlClient().List(
+			ctx,
+			&list,
+			client.InNamespace(cr.Namespace),
+			client.MatchingLabels{label.Cluster: key.ClusterID(&cr)},
+			client.MatchingLabels{label.MachineDeployment: cr.Name},
+		)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		if len(list.Items) != 1 {
+			return nil, microerror.Maskf(executionFailedError, "expected 1 CR got %d", len(list.Items))
+		}
+
+		md = list.Items[0]
+	}
+
+	var forceCGroupsV1 bool
+	{
+		if _, ok := md.Annotations[annotation.NodeForceCGroupsV1]; ok {
+			forceCGroupsV1 = true
+		} else {
+			forceCGroupsV1 = false
+		}
+	}
```

Comment:
```suggestion
	_, forceCGroupsV1 := md.Annotations[annotation.NodeForceCGroupsV1]
```

Reply:
Minor suggestion to save lines. I don't mind either way.

Reply:
I was thinking about this, but it might be harder to read, but no strong feelings